<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="spinBoxDim">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>211</width>
      <height>42</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxSamples">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>211</width>
      <height>42</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Dim</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Samples</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCorrelations">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Correlations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonImpacts">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Impacts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDispersions">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Dispersions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTranslate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxTranslate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>181</width>
      <height>42</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFeatures">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Features</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNorm">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Normalize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOriginal">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPresent">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>103</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Present</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
